src/interfaces/agm-ext-eval-interface.ts

```diff
@@ -25,7 +25,11 @@ export const EmptyAgmExtEval: AgmExtEvalIF = {
   incorporationDate: null,
   isFirstAgm: null,
   agmYear: null,
+  prevAgmDate: null,
+  isPrevExtension: null,
+  prevExpiryDate: null,
+  intendedAgmDate: null,
```

Comment:
Needed to explicitly set values for the `EmptyAgmExtEval`, otherwise the date pickers don’t update the dates properly. I think it is likely because if you `@Watch('data.intendedAgmDate')` and `data.intendedAgmDate` is undefined, when it initially gets set it doesn't seem to trigger the watcher.

Reply:
You are correct. If a variable is undefined, it's not reactive. It's like it doesn't exist (which is kind of the definition of undefined 😀 ).

Reply:
If `prevAgmDate`, `isPrevExtension`, `prevExpiryDate` and `intendedAgmDate` are no longer optional then please remove the "?"s in the interface above. Thx.

Reply:
LGTM